git-merge-pr

```diff
@@ -115,6 +116,7 @@ run_command "git merge -S --no-ff -m $MERGE_MESSAGE origin/$BRANCH_TO_MERGE" ||
 echo "Merge Successful!"
 
 echo "Attempting to push to remote repository 'origin'"
+echo "2 GPG signatures required to clearsign RSL push entry and sign RSL commit"
```

Comment:
Again, we might want to preface so folks know there's an action required... I don't actually expect them to read...